doc/system_requirements.md

```diff
@@ -0,0 +1,77 @@
+# System Requirement Specification Exasol Test Container
+
+## Introduction
+
+Exasol's Test Database Builder for Java (short "TDDB") is a library that helps authors of integration test against databases writing compact and readable test code.
+
+**&#9888; Important:** The TDDB is meant to be used in tests only. It is neither designed nor particularly suited to be used in production scenarios!
+
+## About This Document
+
+### Target Audience
+
+The target audience are end-users, requirement engineers, software designers and quality assurance. See section ["Stakeholders"](#stakeholders) for more details.
+
+### Goal
+
+TDDB It is targeted at functional testing mainly with the central design goal to make writing and reading the test code as convenient as possible for developers.
+
+### Quality Goals
+
+TDDB's main quality goals are in descending order of importance:
+
+1. Compact test code
+1. Readable test code
+
+## Stakeholders
+
+### Integrators
+
+Integrators integrate their solution with databases (like Exasol). To test this, they need to create objects in those databases and populate them with test data.
+
+### Terms and Abbreviations
+
+The following list gives you an overview of terms and abbreviations commonly used in TDDB documents.
+
+* ...
+
+## Features
+
+Features are the highest level requirements in this document that describe the main functionality of TDDB.
+
+### Creating Database Objects
+`feat~creating-database-objects~1`
+
+TDDB allows developers to programmatically create database objects.
```

Comment:
Should we list somewhere the list of objects? It sounds quite wide now.

Reply:
For this level of detail you have requirements of artifact type `req`.